ecommerce/api.py

```diff
@@ -115,6 +144,9 @@ def apply_user_discounts(request):
 
     Args:
         - user (User): The currently authenticated user.
+
+    Returns:
+        True
```

Comment:
does it always return True? :D

Reply:
it really doesn't return anything.. not sure why I set it to True but fixed that in https://github.com/mitodl/mitxonline/pull/708/commits/7ceb1391089ac4e01660d9204101bdddb1127f42